docs/source/about/advanced_usage.rst

```diff
@@ -843,7 +843,7 @@ nv_preset
 **Description**
    The encoder preset to use.
 
-   .. Note:: This option only applies when using nvenc `encoder`_.
+   .. Note:: This option only applies when using nvenc `encoder`_. More information on the presets are available at: https://docs.nvidia.com/video-technologies/video-codec-sdk/nvenc-preset-migration-guide/
```

Comment:
Two things. First, please don't extend beyond 120 characters (for readability in the IDE).

Then the link can be formatted in a prettier way.

```rst
   .. Note:: This option only applies when using nvenc `encoder`_. For more information on the presets, see
      `nvenc preset migration guide <https://docs.nvidia.com/video-technologies/video-codec-sdk/nvenc-preset-migration-guide/>`_.
```